src/themes/default/pages/Home.vue

```diff
@@ -72,57 +72,44 @@ export default {
     // Load personal and shipping details for Checkout page from IndexedDB
     this.$store.dispatch('checkout/load')
   },
-  beforeMount () {
+  async beforeMount () {
```

Comment:
result fo resolved merge conflicts @patzick @pkarw please check if everything is correct

Reply:
for me it looks okay :)